src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.csproj

```diff
@@ -13,5 +13,11 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard2.0'">
```

Comment:
Don't do this.  API compat should be getting the right references.  If its not fix the references don't hack around with content.